lib/puppet/provider/zabbix_proxy/ruby.rb

```diff
@@ -9,6 +9,7 @@ def create
 
     # Set some vars
     host = @resource[:hostname]
+    mode = @resource[:mode]
```

Comment:
I had the same problem when installing a zabbix server. This patch solved it but I had to change line 12 to 'mode = @resource[:mode].to_i'.